### DISCUSSION TITLE
The weight and dimensions of the product is not showing in UI for shop manager

### DISCUSSION THREAD

**Discussion by namlv94** [etsy/open-api#943] [Q&A]
I am using API to create new product and it was successful. Currently when I check that product using getListing API, I notice the product already has weight and dimensions. However when I go to the product edit page on shop manager the values are not showing. Has anyone had a similar problem and has a workaround?
Please see attached pictures:
https://prnt.sc/1_wmLIMZJjeD
https://prnt.sc/fSEFvLXCpPsB
this is my listing_id: 1441509720

**Comment by Gareth064** [comment]
Someone else was having a similar issue here https://github.com/etsy/open-api/discussions/729 but it doesn't look like it got resolved. The Etsy Dev who was commenting on it has now left.

I would recommend you reach out to the Etsy Devs via email for help.

**Comment by namlv94** [comment]
@etsyachristensen Could you please take a look at my problem? Really grateful!

**Reply by Gareth064** [reply]
He doesn't work for Etsy anymore @namlv94 .

The Etsy Dev team don't really support on here anymore. You need to email them for support.

**Reply by namlv94** [reply]
Oh. I didn't know that. Thank you. I will try to contact Etsy to find out what the problem is
